### PR TITLE
finished merged sorted list

### PR DESCRIPTION
**Pull request by fdang836**
Concatenated two list
Return sorted concatenated list

### CODE REVIEW

File: functions/merge_sorted_lists.py [hunk 1]
```diff
@@ -1,2 +1,4 @@
 def merge_sorted_lists(l1, l2):
-    pass
```
Comment (jlee1414) [inline functions/merge_sorted_lists.py:2]: Why is there a "pass" here

File: functions/merge_sorted_lists.py [hunk 2]
```diff
@@ -1,2 +1,4 @@
...
+    merged = l1 + l2
+
+    return sorted(merged)
```
Comment (kkatottn) [inline functions/merge_sorted_lists.py:4]: Nice job! I like your function.

File: tests/merge_sorted_lists_test.py [hunk 3]
```diff
@@ -1,2 +1,40 @@
...
+    l2 = [1]
+
+    # Act
+    answer = merge_sorted_lists(l1, l2)
+
+    # Assert
+    assert answer == [1, 2]
+
+
+def test_for_1_2_3_and_4_5_6():
```
Comment (kkatottn) [inline tests/merge_sorted_lists_test.py:29]: Good! This is nice test.

File: tests/merge_sorted_lists_test.py [hunk 4]
```diff
@@ -1,2 +1,40 @@
...
+def test_for_1_and_2():
+    # Arrange
+    l1 = [1]
+    l2 = [2]
+
+    # Act
+    answer = merge_sorted_lists(l1, l2)
+
+    # Assert
+    assert answer == [1, 2]
```
Comment (jlee1414) [inline tests/merge_sorted_lists_test.py:14]: This is an awesome test. Great job.

File: tests/merge_sorted_lists_test.py [hunk 5]
```diff
@@ -1,2 +1,40 @@
...
+
+
+def test_for_2_and_1():
```
Comment (jlee1414) [inline tests/merge_sorted_lists_test.py:17]: Spot on

**Review by kkatottn — COMMENTED** [review]
Your code is short and precise. Your tests are also thorough.

File: tests/merge_sorted_lists_test.py (hunk 3 above)
Comment (jlee1414) [inline tests/merge_sorted_lists_test.py:29]: More great work

File: tests/merge_sorted_lists_test.py [hunk 6]
```diff
@@ -1,2 +1,40 @@
...
+    # Arrange
+    l1 = [1, 2, 3]
+    l2 = [4, 5, 6]
+
+    # Act
+    answer = merge_sorted_lists(l1, l2)
+
+    # Assert
+    assert answer == [1, 2, 3, 4, 5, 6]
+
```
Comment (jlee1414) [inline tests/merge_sorted_lists_test.py:39]: This is the end

**Review by jlee1414 — COMMENTED** [review]
Wow, this is the best function I have ever seen in my life. My life has been changed forever. I don't know how I have survived this long without every seeing this code. And your tests... wow. I have no words to even describe the feelings that I am experiencing right now. The euphoria I am riding is so immense that I cannot control myself.

**Pull request merged**